test/extended/networking/egressip.go

```diff
@@ -666,7 +703,10 @@ var _ = g.Describe("[sig-network][Feature:EgressIP][apigroup:operator.openshift.
 			spawnProberSendEgressIPTrafficCheckLogs(oc, externalNamespace, probePodName, routeName, targetProtocol, targetHost, targetPort, numberOfRequestsToSend, numberOfRequestsToSend, packetSnifferDaemonSet, egressIPSet)
 		})
 	}) // end testing to external targets
-})
+},
+	g.Entry("", networkAttachmentConfigParams{networkName: "default", topology: "layer3"}),
+	g.Entry("[OCPFeatureGate:NetworkSegmentation][Feature:UserDefinedPrimaryNetworks]", networkAttachmentConfigParams{name: "l3udn", topology: "layer3", role: "primary", cidr: correctCIDRFamily(newPrivilegedEIPOC(), "30.10.0.0/16", "2014:100:200::0/60")}),
+	g.Entry("[OCPFeatureGate:NetworkSegmentation][Feature:UserDefinedPrimaryNetworks]", networkAttachmentConfigParams{name: "l2udn", topology: "layer2", role: "primary", cidr: correctCIDRFamily(newPrivilegedEIPOC(), "30.10.0.0/16", "2014:100:200::0/60")}))
```

Comment:
It looks to me that these should be skipped for MicroShift. It does not have a `networks.operator.openshift.io` so `GetIPAddressFamily()` fails.
@pacevedom correct me if I'm wrong